classes/Collection.js

```diff
@@ -29,18 +30,17 @@ class Collection {
 
   async create(collectionName) {
     try {
-      const config = new Config(this.accessToken, this.siteName)
-      const { content, sha } = await config.read()
-      const contentObject = yaml.safeLoad(base64.decode(content))
-
-      // TO-DO: Verify that collection doesn't already exist
-
-      contentObject.collections[`${collectionName}`] = {
-        output: true 
+      const config = new CollectionConfig(this.accessToken, this.siteName, collectionName)
+      const contentObject = {
+        collections: {
+          [collectionName]: {
+            output: true,
+            order: [],
```

Comment:
As discussed, we will accept an optional array of pages to add to the collection

Reply:
resolved in 03a42a3